Header header
time last_heard
time last_heard_sys_status
uint16 batt_volt
uint16 cpu_load
int8 batt_remaining

uint32 msg_sent_gcs
uint32 msg_received_gcs
uint32 msg_dropped_gcs
uint32 msg_dropped_uas

